notebooks/MERFISH_Pipeline_-_U2O2_Cell_Culture_-_1_FOV.py

```diff
@@ -184,41 +184,15 @@
 # EPY: START code
 from starfish.pipeline.features.pixels.pixel_spot_detector import PixelSpotDetector
 psd = PixelSpotDetector(
-    codebook='https://s3.amazonaws.com/czi.starfish.data.public/MERFISH/codebook.csv',
+    codebook=codebook,
     distance_threshold=0.5176,
     magnitude_threshold=1,
     area_threshold=2,
-    crop_size=40
+    crop_size=(0, 40, 40)
 )
 
-spot_attributes, decoded = psd.find(s)
-# EPY: END code
-
-# EPY: START code
-spot_attributes.head()
-# EPY: END code
-
-# EPY: START code
-res = decoded.result  # this should be consistent across assays; 
-# this one doesn't have a quality, but it should eventually converge to a shared type
-res.head()
-# EPY: END code
-
-# EPY: START markdown
-# In the above method, the private method of the decoder is used, which exposes additional metadata about the spots. 
-# EPY: END markdown
-
-# EPY: START code
-print('Additional metadata:')
-[f for f in dir(decoded) if not f.startswith('_')]
-# EPY: END code
-
-# EPY: START markdown
-# Spot attributes are stored as skimage RegionProperties attributes
-# EPY: END markdown
-
-# EPY: START code
-decoded.spot_props[:3]
+spot_intensities, region_properties, label_image, intensity_image, decoded_image = psd.find(s.image)
```

Comment:
this is a pretty different API than when you use the GaussianSpotDetector in the ISS pipeline: 

```
p = SpotFinder.GaussianSpotDetector(
    min_sigma=min_sigma,
    max_sigma=max_sigma,
    num_sigma=num_sigma,
    threshold=threshold,
    blobs_stack=blobs_stack,
    measurement_type='mean',
)

# blobs = dots; define the spots in the dots image, but then find them again in the stack.
intensities = p.find(filtered_stack)
```


Perhaps a simpler API would be that all find methods return a 'spot_intensities' table AND an optional named dictionary of key value pairs.